extension/src/content.ts

```diff
@@ -1,6 +1,6 @@
 import type { PlasmoCSConfig } from "plasmo";
 
-import { appLoader } from "~scripting";
+import Kernel from "@Scripting/index";
```

Comment:
This is probably going to be reverted eventually, nobody in the ecosystem uses PascalCase on the path aliases.

Reply:
Also, the preferred way of exporting is not `default` in most of the cases. So we would have:

```suggestion
import { Kernel } from "@scripting";
```

This is probably what we will be aiming at. But this can be changed later